src/Core/Rexl.Harness/Storage.cs

```diff
@@ -61,12 +62,16 @@ public virtual Task<(Link full, Stream stream)> LoadStreamAsync(Link linkCtx, Li
 
     /// <summary>
     /// Storage abstraction providing default methods to read/write file streams
-    /// from/to the local file system.
+    /// from/to the local file system and http.
     /// </summary>
     public abstract class LocalFileStorage : Storage
     {
+        private readonly HttpClient _http;
+
         protected LocalFileStorage()
         {
+            _http = new HttpClient();
```

Comment:
Should LocalFileStorage be disposable and dispose of this HttpClient?

Reply:
Generally, a `LocalFileStorage` is intended to live as long as a `Harness`, which is typically for the lifetime of the host application.